### ISSUE TITLE
Compile command fails with Solidity 0.6+

### ISSUE THREAD

**Issue by spalladino** [OpenZeppelin/openzeppelin-sdk#1406]
Compiling a project with solc 0.6.1:

```bash
$ ozdev compile
✖ Compiling contracts with solc 0.6.1 (commit.e6f7d5a4)
soljson.Pointer_stringify is not a function
```

**Comment by spalladino** [comment]
I'm no longer being able to reproduce this one, either on `2.6.0`, `2.7-rc.1`, or `release/2.7`. @BrendanChou I see you upvoted: can you reproduce this issue?

**Comment by BrendanChou** [comment]
I had a similar issue but it wasn't the same function. I just went back to Solidity 0.5. My impression was that the 2.6.0 release was not even usable with Solidity 0.6?

**Comment by spalladino** [comment]
[This](https://github.com/OpenZeppelin/openzeppelin-sdk/issues/1350) was the main issue with Solidity 0.6 on `2.6.0`, which is [fixed](https://github.com/OpenZeppelin/openzeppelin-sdk/pull/1362) in `2.7-rc.0`. 

Another issue is that `Initializable.sol` is not set up for 0.6, though we'll [fix that](https://github.com/OpenZeppelin/openzeppelin-sdk/pull/1413) in `2.7-rc.2` (to be released this week), which should set everything up for 2.7 to be compatible with Solidity 0.6.

I'll close this issue then, and see if it comes up again during testing of the `2.7-rc.2`. Thanks!

**Comment by BrendanChou** [comment]
Thanks! Appreciate the work you all put into these libraries

**Comment by spalladino** [comment]
Reopening as @nventuro managed to reproduce it on `OpenZeppelin/openzeppelin-contracts#1c72343465bb0a9f5e365061f9a182154a394a34`

**Comment by frangio** [comment]
Was this closed again by accident?

**Comment by spalladino** [comment]
Nope, because it was fixed by #1427, which just got merged into master
